python/src/py_opcua_module.cpp

```diff
@@ -242,10 +242,16 @@ static void Node_SetValue(Node & self, const object & obj, VariantType vtype)
 // UaClient helpers
 //--------------------------------------------------------------------------
 
+template<typename T>
+boost::shared_ptr<T> make_shared_ptr(std::shared_ptr<T>& ptr)
+{
+    return boost::shared_ptr<T>(ptr.get(), [ptr](T*) mutable {ptr.reset();});
+}
```

Comment:
Can you confirm what ``make_shared_ptr`` is doing - a conversion from std::shared_ptr to its boost equivalent?
We should probably replace all boost::shared_ptr to std:: shared_ptr in the wrapper.